src/network/network_coordinator.cpp

```diff
@@ -229,6 +343,118 @@ void ClientNetworkCoordinatorSocketHandler::GetListing()
 	this->SendPacket(p);
 }
 
+/**
+ * Join a server based on an invite code.
+ * @param invite_code The invite code of the server to connect to.
+ * @param connecter The connecter of the request.
+ */
+void ClientNetworkCoordinatorSocketHandler::ConnectToServer(const std::string &invite_code, TCPServerConnecter *connecter)
+{
+	assert(invite_code[0] == '+');
```

Comment:
`assert(StrStartsWith(invite_code, "+"));`?
Mostly for the even rarer case of the invite_code being empty when getting here.